pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/JdbiCompositeAnswer.java

```diff
@@ -69,6 +69,7 @@
                                 parentAnswer.setId(rs.getLong("parent_answer_id"));
                                 parentAnswer.setGuid(rs.getString("parent_answer_guid"));
                                 parentAnswer.setQuestionStableId(rs.getString("parent_question_stable_id"));
+                                parentAnswer.setActivityInstanceId(rs.getLong("parent_activity_instance_id"));
```

Comment:
I'm a bit afraid to touch the composite answer code, so I left this `JdbiCompositeAnswer` file.